config/samples/cluster_v1alpha1_eks.yaml

```diff
@@ -9,7 +9,23 @@ spec:
   crossAccountRoleName: "aws-eks-cluster-controller-management"
   region: "us-test-1"         # region where child EKS cluster need to be created
   controlPlane:
-    clusterName: "eks-1"
+    clusterName: "eks-sample"
   nodeGroups:
-    - {name: "g1"}
-    - {name: "g2"}
\ No newline at end of file
+    - name: "g1"
+      iamPolicies: 
+        - policyName: "root"
+          policyDocument: 
+            version: "2012-10-17"
+            statement: 
+              - effect: "Allow"
+                action: ["*"]
+                resource: ["*"]
+    - name: "g2"
+      iamPolicies: 
+        - policyName: "root"
+          policyDocument: 
+            version: "2012-10-17"
+            statement: 
+              - effect: "Allow"
+                action: ["*"]
+                resource: ["*"]
```

Comment:
Very Minor:
User would add the policy in json; may be easier for user to copy paste rather than converting the keys from Titlecase to lower case when preparing CRD definition.

Reply:
I personally hate mixing json with yaml and worrying about encoding back and forth and """'`"quoting`'`'"""

Reply:
I am fine with it, just have to document explicitly to lower the first character while putting policy inside CRD.

Reply:
The convention is that you use lowercase for the first letter, but I don't think there is anything that requires that.